1_inventory/src/check_characteristics.R

```diff
@@ -96,7 +97,8 @@ find_similar_characteristics <- function(characteristics_select, param_groups_se
     matched_chars_not_selected <- matched_chars[!matched_chars %in% characteristics_select]
     
     header_selected <- "The following characteristics are already included in the WQP codes cfg file: "
-    header_not_selected <- "The following characteristics might be relevant to the selected parameter and are not included in the WQP codes cfg file: "
+    header_not_selected <- paste0("The following characteristics might be relevant to the selected ",
+                                  "parameter and are not included in the WQP codes cfg file: ")
```

Comment:
This is such a cool feature of this pipeline

Reply:
Thanks! 😃  This feature was motivated by questions that came up in DRB IWAAs that boiled down to, _How can I know if I've captured all the different characteristic names?_